jekyll/_cci2/sample-config.md

```diff
@@ -27,8 +27,10 @@ version: 2
 jobs:
   build:
     working_directory: ~/mern-starter
+    # The first image in this list runs on the primary container where all steps are run.
     docker:
       - image: circleci/node:4.8.2
+    # This image is the secondary container image which is run in a common network where ports exposed on the primary    container are available on localhost.   
```

Comment:
Suggest: "The secondary container is an instance of the second listed image which is run in a common network where ports exposed on the primary  container are available on localhost.